propagators/opentelemetry-propagator-grpc-census-binary/test/index-webpack.ts

```diff
@@ -0,0 +1,23 @@
+/*!
+ * Copyright 2020, OpenTelemetry Authors
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      https://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+// This file is the webpack entry point for the browser Karma tests. It requires
```

Comment:
This propagator is not applicable for browser, please remove this file.

Reply:
Thanks @mayurkale22. I will address your comments tomorrow.

Yes, I did verify this propagator (both inbound and outbound gRPC, from/to services instrumented with OpenCensus) as part of the fork of microservices-demo mentioned in the description.

Reply:
Latest commit should hopefully cover your first pass @mayurkale22. (FYI, next "I signed it" comment is related to CLA for CNCF).